chidioguejiofor/Ojumo-Club
# #162083391): initialise repo

#### What does this PR achieve?
- initialises pipenv shell and codebase
- adds a sample test to the app
- adds circleci config to the app
- adds config for pre-commit hook to the app
- ensures that tests are passing with at least 90% coverage before commit
- updates README to be more informative

#### Manual Testing Steps
- Fetch branch by running  `git fetch origin  ch-initialise-codebase-162083391`
- Checkout to the branch via `git checkout  ch-initialise-codebase-162083391`
- Make the bash scripts executable via `chmod x+ ./scripts/install-hook.bash ./scripts/pre-commit.bash ./scripts/run-tests.bash `
- Install the pre-commit hook via `./scripts/install-hook.bash`
- Once you commit you should see the test run and two new files namely `coverage.xml` and `.coverage` should be generated

#### Screenshots:
<img width="900" alt="screenshot 2018-11-27 at 8 31 42 am" src="https://user-images.githubusercontent.com/38565349/49065646-ea618380-f21e-11e8-8ede-e66b78894a4d.png">

#### Any new deployment instructions
- None

#### New environmental Variables?
- None

#### Pivotal Tracker
[#162083391](https://www.pivotaltracker.com/story/show/162083391)
